charts/ztka/templates/NOTES.txt

```diff
@@ -2,16 +2,18 @@
 {{- if .Values.ingress.enabled }}
   Open {{ include "ztka.consoleFQDNWithScheme" . }} in browser.
 {{- else if .Values.deploy.contour.enable }}
-  {{- if (eq (index .Values.contour.envoy.service.annotations "service.beta.kubernetes.io/aws-load-balancer-type" | toString ) "nlb") }}
   Get load balancer address via:
   kubectl get service {{ .Release.Name }}-contour-envoy --namespace {{ .Release.Namespace }} -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'
 
   Add DNS records of following domains such that it resolves to above address:
   - {{ include "ztka.consoleFQDN" . }}
   - {{ include "ztka.coreConnectorFQDN" . }}
   - {{ include "ztka.userFQDN" . }}
-  {{- end }}
+
   Open {{ include "ztka.consoleFQDNWithScheme" . }} in browser.
+
+  Note: If you are using a cluster with no load-balancer, then the address will be "<pending>".
+        If it is Kind or Minikube cluster, check out respective docs to get the external address.
```

Comment:
I believe the output will be empty in case of `kind`. How about we include the command to get the hostname only for aws and for others we just include the section about adding the DNS records?

Reply:
Just DNS records might be confusing for users, it leaves how to get application external ip question unanswered.
As per the above comment `kubectl get service release-contour-envoy -n paralus` will return EXTERNAL-IP (include hostname too) for all the environments except Kind. It will be `<pending>` for Kind cluster. So this `Note` will help.

Reply:
Combining with the change to `k get svc ...`, I guess it makes sense to leave it there. 👍🏼